database/README.md

```diff
@@ -0,0 +1,14 @@
+This document explains the database that the orderbook api uses.
+
+We need to store orders persistently even if the orderbook service goes. For this we use a PostgreSQL database. The schema can be found in `schema.sql`.
+
+In addition to the orders, we also store trade events. This allows us to keep track of filled amounts.
+
+To prevent users from spamming the database we should when receiving an order check that
+* the erc20 spending is approved
+* valid_to is not in the past and not more than <1 year> in the future
+* the order owner does not already have more than <1000> valid orders
```

Comment:
Seems fair. Do we plan to have a cleanup functionality which archives or purges invalid/expired orders?

Reply:
It depends. If we feel the need to clean up space in the database we can delete expired orders or move them to a more long term storage place. Or it might take up so little space that it doesn't matter. I just want to make sure a user cannot place an unlimited amount of orders here.